lib/error.js

```diff
@@ -75,7 +76,11 @@ AppError.translate = function (response) {
     }
   }
   else if (payload.validation) {
-    error = AppError.invalidRequestParameter(payload.validation)
+    if (payload.message && payload.message.indexOf(messages.any.required) >= 0) {
+      error = AppError.missingRequestParameter(payload.validation.keys)
```

Comment:
Actually @rfk, you've made me realise I was making an assumption here. Although `validation.keys` is an array, I hadn't seen Joi populate it with more than one item. Having just checked the readme, I saw [this](https://github.com/hapijs/joi#validatevalue-schema-options-callback):

> * `options` - an optional object with the following optional keys: 
>   * `abortEarly` - when `true`, stops validation on the first error, otherwise returns all the errors found. Defaults to `true`.

We're not setting `abortEarly` so should never see more than one parameter in this array.

Maybe your initial suggestion of naming one parameter in the error was correct after all, what do you think?